rllib/agents/trainer.py

```diff
@@ -2252,12 +2250,13 @@ def should_stop(self, result):
                         and (not min_train_ts or self.trained >= min_train_ts)
                     ):
                         return True
-                # No errors (we got results) -> Break.
-                elif result is not None:
+                    else:
+                        return False
+                # No errors (we got results != None) -> Return True
+                # (meaning: yes, should stop -> no further step attempts).
```

Comment:
based on the comment above, it sounds like exec_plan will handle stopping itself, so we don't need to say it should stop. so we should return False here?

Reply:
Correct, `exec_plan` always goes through the `OncePerTimestepsElapsed` and `OncePerTimeInterval` ops, which handle this logic instead.